apps/authentication-service/src/shared/tokenUtils.ts

```diff
@@ -0,0 +1,69 @@
+import fs from "fs";
+import { decode, JwtPayload } from "jsonwebtoken";
+import os from "os";
+import path from "path";
+
+import { ConsoleColors } from "@/shared/enums";
+
+let token: string | undefined;
+const tokenFileName: string = "brkn-arrw.txt";
+
+export const tokenFile: string = path.join(os.tmpdir(), tokenFileName);
+
+export const getLocalToken = (): string | undefined => {
+  logToken(token);
+  return token;
+};
+
+export const setLocalToken = (newToken: string | undefined): void => {
+  token = newToken;
+};
+
+export const setCachedToken = (token: string): boolean => {
+  try {
```

Comment:
This is swallowing errors. If this is failing, then someone has their configuration wrong.

Reply:
I did that since caching to a file isn't really required. I'm not sure how this relies on their configuration being correct. If it doesn't cache, it could be a system-level issue with node, or permissions or something. 

What's a better way to handle this? Should I throw an error if this doesn't succeed?

Reply:
Yeah I just wouldn't do the try catch. The parent can catch the error, or things can blow up so they know their configuration is wrong